jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -46,26 +46,26 @@ def _get_component(layer):
 
             try:
                 # Retrieve layer data and units
-                data_obj = layer.layer.data.get_component(self.slice_component_label).data
-                data_units = layer.layer.data.get_component(self.slice_component_label).units
+                data_comp = layer.layer.data.get_component(self.slice_component_label)
```

Comment:
Should this be here? 

https://github.com/spacetelescope/jdaviz/pull/2895/files#diff-87fc9fbc017d7ecd743c7868a9c8603d038a7c431ae52f596287a307ee82442dL50-L57

Reply:
I think so, this is just because some changes were made in `main` that weren't backported, and so this brings 3.10 and main into agreement on (most parts) of this method.  Hopefully would make any future backports easier too 🤞